src/colmap/estimators/bundle_adjustment.cc

```diff
@@ -1175,4 +1175,70 @@ void PrintSolverSummary(const ceres::Solver::Summary& summary,
   LOG(INFO) << log.str();
 }
 
+ceres::LossFunction* CreateLossFunctionPerPoint(const std::string& loss_name,
```

Comment:
Seems like a duplicate of `BundleAdjustmentOptions::CreateLossFunction`? Maybe take this out of `BundleAdjustmentOptions` to reuse?